native/cocos/renderer/gfx-base/GFXDef-common.h

```diff
@@ -1213,6 +1213,7 @@ struct Attribute {
 
 using AttributeList = ccstd::vector<Attribute>;
 
+#ifndef SWIGCOCOS
```

Comment:
This file is generated by `native\tools\gfx-define-generator\generate.js`

Reply:
Reverted